test/CouplerState/cplstate_interface.jl

```diff
@@ -1,37 +1,77 @@
 using Test
 using Random
 using ClimaCoupler, Dates, Unitful
+using IntervalSets
+using ClimaCore: Domains, Meshes, Geometry, Topologies, Spaces, Fields, Operators
+
+struct SimulationA{T} <: ClimaCoupler.AbstractSimulation
+    data::T
+end
+ClimaCoupler.name(::SimulationA) = :simA
```

Comment:
I wonder if we could get away with avoiding the (simA, simB, ...) assignment, e.g. using `Symbol`? Again, something we can address during the next iteration. Noted down to discuss.

Reply:
Can you elaborate?

Reply:
I guess it would be good for this to work for an arbitrary number of sims, each of which has it's own user-defined tag. Maybe the solution will become clearer next week, when we focus on the interface! :)